guides/functor-combinator-instances-and-class1s.md

```diff
@@ -0,0 +1,83 @@
+# What
+
+CLC has approved the [proposal #10](https://github.com/haskell/core-libraries-committee/issues/10), to relax instances for Functor combinators and put superclasses on \<class>1 to make the change less-breaking.
```

Comment:
Please put an explicit list of classes affected.

Reply:
Done